admin-dev/themes/default/template/controllers/attribute_generator/content.tpl

```diff
@@ -95,7 +95,7 @@
 
 				<div class="alert alert-info">{l s='You\'re currently generating combinations for the following product:'} <b>{$product_name|escape:'html':'UTF-8'}</b></div>
 
-				<div class="alert alert-info"><strong>{l s='Step 1: On the left side, select the attributes you want to use (Hold down the "Ctrl" key on your keyboard and validate by clicking on "Add")'}</strong></div>
+				<div class="alert alert-info"><strong>{l s='Step 1: On the left side, select the attributes you want to use (Hold down the "Ctrl" key on your keyboard and validate by clicking on "%add_label%")' sprintf=['%add_label%' => {l s='Add' d='Admin.Actions'}]}</strong></div>
```

Comment:
It might be time to offer this string a proper domain, aha! I suggest _Admin.Catalog.Notification_, what do you think?

Reply:
This makes sense, but I believe this template is actually not used any more 🤔 So is it worth changing the domain (maybe even the wording?)

Reply:
What do you mean by 'this template is not used anymore'? this string is not used anymore? or in a different way?

Reply:
The controller has been migrated but the legacy controller and its templates are still present Basically the code is still in the repo but it's not displayed any more

Reply:
Is it still in the translation catalog? If yes, let's localize it in _Admin.Catalog.Notification_. If not, let's keep it as is. :-)

Reply:
It's still in the catalog yes, in `messages` You think it's better to put it in `Admin.Catalog.Notification` ? Because we will have two versions of the message in two domains, even though none are actually displayed

Reply:
But since we create new wording with placeholders anyway maybe it makes sense to use a new domain 🤔

Reply:
I totally agree!